### PR TITLE
Add clickable nodes

### LINKED ISSUE #65: Add radio button to select what click (taptool) does
You should be able to click to trigger selecting pollution history for a node OR choose the pollution start node with a click

### LINKED ISSUE #77: Reintroduce taptool to select node to show history
Also do #65

### PR DESCRIPTION
**Pull request by edwardchalstrey1**
Closes #77 
Closes #65

### CODE REVIEW

File: water/main.py [hunk 1]
```diff
@@ -373,28 +399,35 @@ def plot_bounds(locations):
...
                               button_type="primary", menu=speed_menu)
     speed_dropdown.on_change('value', update_speed)
 
     # Create a div for the timer
     timer = Div(text="")
 
+    # Create a radio button to choose what clicking a node does
+    what_click_does = RadioGroup(
+        labels=["Clicks Choose Pollution History",
+                "Clicks Choose Pollution Injection"],
```
Comment (JimMadge) [inline water/main.py]: These names a bit long-winded

File: water/modules/html_formatter.py [hunk 2]
```diff
@@ -29,7 +29,7 @@ def pollution_location_html(injection_node, injection_color):
 def node_type_html(type="None", color="black"):
     if type == "None":
         color = "black"
```
Comment (JimMadge) [inline water/modules/html_formatter.py]: This is a nice feature

File: water/main.py [hunk 3]
```diff
@@ -134,6 +134,23 @@ def update_node_highlight(attrname, old, new):
...
+        node_count = -1
+        for node in G.nodes():
+            node_count += 1
+            if node_count == first_clicked_node_int:
+                clicked_node = node
+                break
+        if what_click_does.active == 0:
+            pollution_history_select.value = clicked_node
+        if what_click_does.active == 1:
+            pollution_location_select.value = clicked_node
```
Comment (JimMadge) [inline water/main.py]: Can we have move informative labels that 0 and 1.

It looks like these must be integers, but something like
```python
POLLUTION_HISTORY = 0
POLLUTION_LOCATION = 1

...

what_click_does.active == POLLUTION_HISTORY:
...
```
would be more readable.

**Comment by edwardchalstrey1** [comment]
@JimMadge ended up copying the dictionary approach you used for the speed menu which I think looks neat for the click_options too

**Pull request merged**